### PR TITLE
Add keys and separator options

### PR DESCRIPTION
**Pull request by dstiert**
Added the keys and separator params on K/V GET

### CODE REVIEW

File: lib/diplomat/kv.rb [hunk 1]
```diff
@@ -141,5 +145,13 @@ def recurse_get(options)
     def dc(options)
       if options && options[:dc] then use_named_parameter("dc", options[:dc]) else [] end
     end
+    
+    def keys(options)
+      if options && options[:keys] then ['keys'] else [] end
+    end
+    
+    def separator(options)
+      if options && options[:separator] then use_named_parameter("separator", options[:separator]) else [] end
```
Comment (piavlo) [inline lib/diplomat/kv.rb]: should also check that keys option was set to true

File: lib/diplomat/kv.rb [hunk 2]
```diff
@@ -12,6 +12,8 @@ class Kv < Diplomat::RestClient
     # @option options [Boolean] :recurse If to make recursive get or not
     # @option options [String] :consistency The read consistency type
     # @option options [String] :dc Target datacenter
+    # @option options [String] :keys Only return key names.
```
Comment (piavlo) [inline lib/diplomat/kv.rb]: [Boolean]

File: lib/diplomat/rest_client.rb [hunk 3]
```diff
@@ -76,6 +76,7 @@ def parse_body
 
     # Get the key/value(s) from the raw output
     def return_value(nil_values=false)
```
Comment (piavlo) [inline lib/diplomat/rest_client.rb]: consul ?keys returns array of keys, so @raw should be ruby array , not sure i understand how it's properly handled by return_value method?

Comment (dstiert) [inline lib/diplomat/rest_client.rb]: @piavlo: By providing the keys param, you're asking for an array of strings (they keys). So if @raw is an array of strings this just returns that.

**Comment by PeterFCaswell** [comment]
I think this should really be cleaned up in the theoretical new version of the KV object. But I have no objections if you want to move forward with this in the meantime.

@dstiert Could you bump the 0.14.x version to 0.14.x+1, please?

**Comment by dstiert** [comment]
@PeterFCaswell Bumped the version. Appreciate this moving forwards, working on a gem that I need this in to release. Thanks.

**Pull request merged**